modules/web/server/src/main/scala/navigate/web/server/security/FreeLDAPAuthenticationService.scala

```diff
@@ -24,11 +24,11 @@ object FreeLDAPAuthenticationService {
   // Operations on ldap
   object LdapOp {
```

Comment:
Are we going to use LDAP authentication in navigate?

Reply:
Whatever we use for Observe, we use for Navigate